tools/docs/config.ts

```diff
@@ -142,13 +145,55 @@ function genTable(obj: [string, string][], type: string, def: any): string {
 }
 
 function stringifyArrays(el: Record<string, any>): void {
+  const ignoredKeys = ['default', 'experimentalIssues'];
+
   for (const [key, value] of Object.entries(el)) {
-    if (key !== 'default' && Array.isArray(value)) {
+    if (!ignoredKeys.includes(key) && Array.isArray(value)) {
       el[key] = value.join(', ');
     }
   }
 }
 
+function genExperimentalMsg(el: Record<string, any>): string {
+  const ghIssuesUrl = 'https://github.com/renovatebot/renovate/issues/';
+  let warning =
+    '\n<!-- prettier-ignore -->\n!!! warning "This feature is flagged as experimental"\n';
+
+  if (!el.experimental) {
+    return '';
+  }
+
+  if (el.experimentalDescription) {
+    warning += indent`${2}${el.experimentalDescription}`;
+  } else {
+    warning += indent`${2}Experimental features might be changed or even removed at any time.`;
+  }
+
+  const issues: string[] = [];
+  for (const issue of el.experimentalIssues) {
```

Comment:
```suggestion
  for (const issue of el.experimentalIssues ?? []) {
```
typescript null check